packages/wonder-blocks-core/util/util.js

```diff
@@ -59,8 +60,22 @@ export function processStyleList<T: Object>(
         }
     });
 
+    const inlineStylesObject = Object.assign({}, ...inlineStyles);
+
+    // TODO(somewhatabstract): When aphrodite no longer puts "!important" on
+    // all the styles, remove this <ADD JIRA ISSUE HERE IF THIS PASSES REVIEW>
+    // If we're not snapshotting styles, let's create a class for the inline
+    // styles so that they can apply to the element even with aphrodite's
+    // use of !important.
+    if (inlineStyles.length > 0 && !shouldInlineStyles) {
+        const inlineStylesStyleSheet = StyleSheet.create({
+            inlineStyles: inlineStylesObject,
+        });
+        stylesheetStyles.push(inlineStylesStyleSheet.inlineStyles);
+    }
```

Comment:
This is the code that creates the extra class for us.

Reply:
Note that it is pushed onto the end of the stylesheet list so that just as with inline styles, these take precedence over other classes.

Reply:
This may have perf implications, but I know were doing similar things elsewhere in the codebase so maybe not it's not that bad.

Reply:
I did some basic perf checks and didn't see anything that was so horrible as to push back on this. If there are, perhaps we can have an escape hatch and make this inline->class thing optional?

Reply:
Like, if the style has a `_noClassConversion` flag set to `true`, we don't do this bit.

Reply:
That way for animations, we can opt out.

Reply:
Cool.  I also checked on slack and Emily and Matt aren't concerned about the perf implications.